GliaWidgetsTests/Glia.Environment.Failing.swift

```diff
@@ -73,7 +73,7 @@ extension Glia.Environment {
         dismissManager: .init { _, _, _ in
             fail("\(Self.self).dismissManager")
         },
-        combineScheduler: .failing
+        combineScheduler: .live
```

Comment:
Is it expected to use `.live` in unit tests?

Reply:
No, this is a mistake

Reply:
fixed [d09cfb9](https://github.com/salemove/ios-sdk-widgets/pull/1246/commits/d09cfb9787623de377d690cc4bb87ef0b82abd13)